platform/darwin/src/MGLNetworkConfiguration.h

```diff
@@ -23,12 +49,14 @@ MGL_EXPORT
  If this property is set to nil or if no session configuration is provided this property
  is set to the default session configuration.
  
- Assign this object before instantiating any `MGLMapView` object.
+ Assign this object before instantiating any `MGLMapView` object, or using
+ `MGLOfflineStorage`
```

Comment:
What’s an example of a place where this object can be assigned safely? Consider that fixing #227 may require us to invoke `+[MGLOfflineStorage sharedStorage]` in an `+initialize` method.

Reply:
Ugh. Let's address that when we fix #227. Hopefully we can find a way around using an `+initialize`.